modules/openssl

```diff
@@ -8,7 +8,7 @@ openssl_url := https://www.openssl.org/source/$(openssl_tar)
 openssl_hash := 6c13d2bf38fdf31eac3ce2a347073673f5d63263398f1f69d0df4a41253e4b3e
 
 # hack to provide path to libgcc
-LIBGCC_DIR := $(dir $(shell $(heads_cc) -print-libgcc-file-name))
+LIBGCC_DIR := $(dir $(shell $(heads_cc) -print-libgcc-file-name 2>/dev/null))
```

Comment:
The variable is unused and from the error it sounds like this shell command fails entirely.  Just remove it (and the comment) :slightly_smiling_face:

Reply:
@JonathonHall-Purism fixed under [fa063bf](https://github.com/linuxboot/heads/pull/1680/commits/fa063bf44123b720b5a1efd017b453e0687a5016) updated OP.